Cargo.toml

```diff
@@ -8,11 +8,13 @@ edition = "2018"
 anyhow = "1.0.40"
 cpal = "0.13.3"
 ringbuf = "0.2.4"
+serde = { version = "1.0.125", features = ["derive"] }
+serde_yaml = "0.8.17"
```

Comment:
[Serde](https://serde.rs/) is a JSON + other format parsing library. By far the nicest one I've used in any language.